packages/frontend/src/build/ApiActivity.ts

```diff
@@ -25,11 +25,37 @@ const getMockData = (): ApiActivity => {
     data.push([START.add(i, 'days'), 100_000 + i * 10_000])
   }
 
+  const arbitrumData: [UnixTime, number][] = data.map((d) => [d[0], d[1] * 0.6])
+  const optimismData: [UnixTime, number][] = data.map((d) => [d[0], d[1] * 0.3])
+  const dydxData: [UnixTime, number][] = data.map((d) => [d[0], d[1] * 0.1])
+
   return {
     combined: {
       types: ['timestamp', 'daily tx count'] as ['timestamp', 'daily tx count'],
```

Comment:
Are those necessary? AFAIK Typescript is smart enough to figure this out
```suggestion
      types: ['timestamp', 'daily tx count'],
```